integration/container_vulnerability_test.go

```diff
@@ -31,10 +31,8 @@ import (
 )
 
 const (
-	registry   = "index.docker.io"
-	repository = "lacework/lacework-cli"
-	tag1       = "ubuntu-1804"
-	tag2       = "debian-10"
+	registry        = "index.docker.io"
+	cleanRepository = "techallylw/test-cli-clean"
```

Comment:
How so you are using the clean docker image? I see it does have one vulnerability: 

https://4451-242235794-gh.circle-artifacts.com/0/circleci-artifacts/techallylw-test-cli-clean-sha256%3A7bf7cef98cceeb038437e7af713153ea28e93a00875d11f284b6f7c6dcb26ba9.html

Two questions:
* Shouldn't we need to assert that the clean image has NO vulnerabilities? (but it does 🤔)
* Shouldn't we be using the dirty image here instead?

Reply:
Only if the tests expect vulns we should use dirty container. It was vuln free initially. Having something guaranteed to be vuln free will be difficult. Best we can do is hope that patches happen quickly for new CVE's. Perhaps there's a base image with a focus on quickly patching new cve's?

Reply:
But you're right for 1 of the tests that rely on the assessment output from this scan expect vulns. So we should switch to to the dirty container.